### PR TITLE
Documentation fix for downstream_url

### PR DESCRIPTION
**Pull request by nileshcs**
**What this PR does / why we need it**:

Documentation of _downstream_url_ is outdated as it references Prometheus instead of Loki, this PR updates it. downstream_url needs to contain the protocol (http://) where-in the example incorrectly mentions only the hostname and port; which is fixed in this PR.

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/sources/configuration/query-frontend.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ data:
 
     frontend:
       log_queries_longer_than: 5s
-      downstream_url: querier.<namespace>.svc.cluster.local:3100
+      downstream_url: http://querier.<namespace>.svc.cluster.local:3100
```
Comment (sandeepsukhani) [inline docs/sources/configuration/query-frontend.md:63]: Not that its a problem setting a scheme explicitly here but I wonder if you are facing any issue without setting the it. We do not face any issue when it is not set.

**Review by sandeepsukhani — APPROVED** [review]
Just added a small comment. It LGTM.
Thanks for the PR!

File: docs/sources/configuration/query-frontend.md (hunk 1 above)
Comment (nileshcs) [reply docs/sources/configuration/query-frontend.md:63]: Yes, I just tested it. Without http://, it gives the following error  regarding unsupported protocol scheme

```
level=warn ts=2021-01-04T14:37:38.35781553Z caller=logging.go:62 traceID=8c074c80cfcc1a msg="GET /ready (500) 156.572µs Response: \"unsupported protocol scheme \\\"loki.service.tr3.consul\\\"\\n\" 
```

**Review by owen-d — APPROVED** [review]
LGTM, thanks for this!

**Pull request merged**